source/Octopus.Tentacle.Tests/Integration/ScriptServiceV2Fixture.cs

```diff
@@ -507,7 +507,7 @@ void WriteLogsToConsole(List<ProcessOutput> logs)
         {
             foreach (var log in logs)
             {
-                TestContext.Out.WriteLine(log.Text);
+                TestContext.Out.WriteLine("{0:yyyy-MM-dd HH:mm:ss K}: {1}", log.Occurred.ToLocalTime(), log.Text);
```

Comment:
I am surprised that this (the old code) is not using the Logger from the base class. The logger is created by `SerilogLoggerBuilder` which can just be `new`ed up at any time if required.

Reply:
🤷 yeah I'm not sure, I'm not overly familiar with what is available in the tests here, I'll take a look though, thanks.

Reply:
If you end up making it go to the seri log Logger, that change can be merged in as it would be helpful.

Reply:
To close the loop this is not integration tests and so can not use the serilog builder I mentioned above.